Description: gr-wavelet fails with cmake 3.4.1 giving
 Could NOT find GSL (missing:  GSL_INCLUDE_DIRS GSL_LIBRARY_DIRS)
 without this workaround. Earlier invocation by gr-fec succeeds,
 so likely some problem with use of CMake cache by FindGSL.cmake.
Author: A. Maitland Bottoms <bottoms@debian.org>

--- a/cmake/Modules/FindGSL.cmake
+++ b/cmake/Modules/FindGSL.cmake
@@ -145,5 +145,5 @@
 
 ENDIF(GSL_FOUND)
 
-INCLUDE(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(GSL DEFAULT_MSG GSL_LIBRARIES GSL_INCLUDE_DIRS GSL_LIBRARY_DIRS)
+#INCLUDE(FindPackageHandleStandardArgs)
+#FIND_PACKAGE_HANDLE_STANDARD_ARGS(GSL DEFAULT_MSG GSL_LIBRARIES GSL_INCLUDE_DIRS GSL_LIBRARY_DIRS)
